sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -928,7 +928,7 @@ def __iter__(self):
         # return base64 encoded bytes as byte type on python 3
         # which matches the behavior of Beam Java SDK
         for i in range(len(row.f)):
-          if self.schema.fields[i].type == 'BYTES':
+          if self.schema.fields[i].type == 'BYTES' and row.f[i].v:
```

Comment:
Thanks a lot! Could you please add to one of BQ IT tests a test scenario that reads and writes a row of None values for BQ datatypes that allow value to be None ?

Thank you!